rocketpy/simulation/monte_carlo.py

```diff
@@ -0,0 +1,726 @@
+import ast
+from time import process_time, time
+
+import numpy as np
+import simplekml
+
+from rocketpy.plots.monte_carlo_plots import _MonteCarloPlots
+from rocketpy.prints.monte_carlo_prints import _MonteCarloPrints
+from rocketpy.simulation.flight import Flight
+from rocketpy.tools import (
+    generate_monte_carlo_ellipses,
+    generate_monte_carlo_ellipses_coordinates,
+)
+
```

Comment:
"most RocketPy import statements are relative, but these are absolute."

Are you sure about that?

My impression is that relative imports are no longer the majority. I think we are using them only in a few `__init__` files.

All in all we will never change the module name from "rocketpy" to other, so that's not a problem.

Reply:
I checked some earlier PRs in which we discussed that and I have "misremembered" #415, you are correct.

Nonetheless, my comment was mostly to make sure we progress knowingly the import type we are using, since both ways work.

Reply:
Personal opinion: I would open a PR changing all the remaining relative imports to absolute imports.
The reason: I think the code is more readable, a new user trying to read the code source would benefit from that, while experienced developers wouldn't notice the difference.